editor/components/block-settings-menu/block-transformations.js

```diff
@@ -27,9 +27,12 @@ function BlockTransformations( { blocks, small = false, onTransform, onClick = n
 	return (
 		<Fragment>
 			<div className="editor-block-settings-menu__separator" />
+			<span
+				className="editor-block-switcher__menu-title"
```

Comment:
Are you using this class for something? We try not to inherit styles by reusing classes.

Reply:
I did use it to inherit, indeed. Pushed a change which simply duplicates the styles, but I'm not sure that's the best way to do it either. 

It seems like the More Menu, and the Block More Menu (block ellipsis menu) and the Block Switcher should all use a generic Menu component. But I'm not sure I have the skills necessary to make that happen, or whether it should happen in this PR. If yes then feel free to push to this branch.